src/storage/txn/flow_controller.rs

```diff
@@ -334,7 +359,12 @@ where
 // And all of them are collected from the hook of RocksDB's event listener.
 struct CFFlowChecker {
     // Memtable related
-    last_num_memtables: Smoother<u64, 20>,
+    // Assume the write flow is about 100MB/s, since the memtable size of
+    // write & default cf is 128MB, so there are about 300 flush events in 5
+    // minutes, we will observe the number of memtables about 300 times. We
+    // set the capacity of `last_num_memtables` to 512 to make sure we keep
+    // enough fresh record in the most cases.
+    last_num_memtables: Smoother<u64, 512>,
```

Comment:
large capacity would make the average of num_memtables lagging, which can't avoid the accumulation of memtables in time.